src/components/figures/figure/figure.xml

```diff
@@ -1,67 +1,75 @@
 <templates>
   <t t-name="o-spreadsheet-FigureComponent" owl="1">
-    <div class="o-figure-wrapper" t-att-style="getContainerStyle()">
-      <div class="o-figure-overflow-wrapper">
-        <div
-          class="o-figure"
-          t-on-mousedown.stop="(ev) => this.onMouseDown(ev)"
-          t-att-style="getFigureStyle()"
-          t-att-class="{active: isSelected, 'o-dragging': dnd.isActive}"
-          t-ref="figure"
-          tabindex="0"
-          t-on-keydown.stop="(ev) => this.onKeyDown(ev)"
-          t-on-keyup.stop="">
-          <t
-            t-component="figureRegistry.get(this.props.figure.tag).Component"
-            t-key="this.props.figure.id"
-            sidePanelIsOpen="props.sidePanelIsOpen"
-            onFigureDeleted="props.onFigureDeleted"
-            figure="displayedFigure"
-          />
+    <div
+      class="o-figure-container-new position-absolute pe-none"
+      t-att-class="{'overflow-hidden': !dnd.isActive}"
+      t-att-style="gontainerStyle">
+      <div
+        class="o-figure-anchor w-0 h-0 overflow-visible position-absolute"
```

Comment:
it is not. We could probvably rename those anchors with a proper name which matches their use - like `resizer`

The idea of the 0 sized div is to have a "hook" (maybe better than anchor ?) on which we place the actual figure part. It's an invisible point of reference and when we scroll, we move that point of reference.  since it contains the figure, it cannot be invisible, so 0 size^^